man/bundle-config.ronn

```diff
@@ -176,8 +176,6 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
    When set, Gemfiles containing multiple sources will produce errors
    instead of warnings.
    Use `bundle config unset disable_multisource` to unset.
-* `disable_platform_warnings` (`BUNDLE_DISABLE_PLATFORM_WARNINGS`):
```

Comment:
Is there any way how to inform users using this setting that it is not valid anymore and can be removed?

Reply:
I've been planning to work on setting validation & deprecation for a while, but I haven't yet found the time. For the time being, these non-existing settings are just ignored, which doesn't sound like a specially bad behavior. But I plan to improve on this in the future :+1:.